plugins/outputs/health/README.md

```diff
@@ -75,6 +75,14 @@ the field is not found on a metric no comparison will be made.
 
 Comparisons must be hold true on all metrics for the check to pass.
 
+The available comparison operators are:
+ - `gt` greater than
+ - `ge` greater than or equal to
+ - `lt` less than
+ - `le` less than or equal to
+ - `eq` equal to
+ - `ne` not equal to
```

Comment:
Please fix the linter issues! There should be no indentation and there must be an empty line before and after the list!

Reply:
@srebhan fixed